fields-core/src/commonMain/kotlin/dev/icerock/moko/fields/core/FormFieldExt.kt

```diff
@@ -4,5 +4,6 @@
 
 package dev.icerock.moko.fields.core
 
-fun List<FormField<*, *>>.validate(): Boolean =
-    all { it.validate() }
+fun List<FormField<*, *>>.validate(): Boolean {
+    return map { it.validate() }.all { true }
```

Comment:
you not compare value with true. so in all cases you will receive true as result

Reply:
fixed